mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -320,6 +362,23 @@ class KMeans @Since("1.5.0") (
       .setMaxIterations($(maxIter))
       .setSeed($(seed))
       .setEpsilon($(tol))
+
+    if (isDefined(initialModel)) {
+      // Check the equal of dimension
```

Comment:
very minor, but this doesn't make sense. Maybe `// Check that the feature dimensions are equal`